modules/shared/server/controllers/mail.controller.js

```diff
@@ -0,0 +1,69 @@
+const nodemailer = require('nodemailer');
+const aws = require('aws-sdk');
+
+const config = require('../../../../config/config');
+
+const sesApiVersion = '2010-12-01';
+
+function MailController(logger) {
+  const knownProviders = [
+    'ses'
+  ];
+
+  /**
+   * send and email using aws-ses
+   * @param {Object} params
+   * @param {String} params.text - text of email
+   * @param {String} params.from - email sent from
+   * @param {String} params.to - email sent to
+   * @param {String} params.subject - subject of email
```

Comment:
This isn't actually how jsDoc works I don't think.  Since only one parameter is enumerated in the function definition, there's only one to get declared in the doc.  I'd have to double check the jsDoc info, but I think the best way to document what this object looks like is to either a) put it between the one liner at the top of jsdoc  and the parameter list as a note, or b) declare a type to handle the complex object and have that be defined as the type.

Obviously a type would be a bit subjective in JS, but you can still define a function that returns the base fields that make up a "type".